app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PasswordResetsController < ApplicationController
+  def show
+    user = User.where(token: params[:id]).first
```

Comment:
You should avoid using `where(...).first` when retrieving 1 object.

HOMEWORK: Explain why. Hint: take a look at the lesson 1.